share/wordlists/eff_large.wordlist

```diff
@@ -7742,12 +7742,12 @@ yiddish
 yield
 yin
 yippee
-yo-yo
+yes
```

Comment:
This is out of alphabetical order. The rest of the list seems alphabetical.

Reply:
Whoops. Let me fix that. I was looking at a shorter sub-list.